lib/improver/tests/utilities/test_cli_utilities.py

```diff
@@ -29,13 +29,23 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 """Unit tests for utilities.cli_utilities."""
-
+import json
+import os
 import unittest
+from subprocess import call as Call
+from tempfile import mkdtemp
+
+import iris
+import numpy as np
 
-from improver.utilities.cli_utilities import radius_or_radii_and_lead
+from improver.tests.set_up_test_cubes import set_up_variable_cube
+from improver.utilities.cli_utilities import (radius_or_radii_and_lead,
+                                              load_json_or_none,
+                                              load_cube_or_none)
+from improver.utilities.save import save_netcdf
 
 
-class TestRadiusOrRadiiAndLead(unittest.TestCase):
+class Test_radius_or_radii_and_lead(unittest.TestCase):
```

Comment:
Comments for the tests below.

test_both_None: The assert test should check the exception message.

test_radii_and_lead: "second index" do you mean second argument?

test_both_used: Should the function being tested not raise a warning if it is getting all of the inputs? The user may not realise both are set, and will not know what is actually being used.